docs/introduction/prerequisites.md

```diff
@@ -0,0 +1,454 @@
+### Can You Develop on Windows?
+It is possible to use Windows for development, but there are important considerations to keep in mind. Since ESO's development environment is not optimized for Windows, compatibility issues may arise with tools like Make, Tilt, and shell scripts. The project's automation scripts and commands are written for Unix environments, using bash scripting, which might not be compatible with Windows without adaptations. This tutorial will not cover the installation and configuration of tools on Windows due to its complexity and lack of testing.
+
+---
+
+## Install Go (Golang)
+
+<details>
+  <summary>About Golang</summary>
+  <h3> What is Go?</h3>
+  <p> Go, also known as Golang, is a programming language created by Google. It is known for being efficient, easy to learn, and excellent for developing fast and scalable applications.</p>
+  <h3> Why is Go needed?</h3>
+  <p> In the <strong>External Secrets Operator</strong> project, Go is used to develop core parts of the code. It is required to compile, run, and contribute to the project's source code.</p>
+</details>
+
+<details>
+  <summary>Golang Installation</summary>
+  <h3> Required Version</h3>
+  <p><strong>Minimum version:</strong> Go 1.20 or higher.</p>
+  <p><strong>Recommended version:</strong> Go 1.23.3</p>
+  <blockquote> As of this writing, the latest version of Go is <strong>1.23.3</strong>As of this writing, the latest version of Go is 1.23.3 , which worked perfectly with the <strong>External Secrets Operator</strong> project. Previous versions failed to test the application. Before testing the project, check your Go version.</blockquote>
+
+  <h3> How to Install Go</h3>
+
+  <details>
+    <summary>Installation on Linux</summary>
+    <h3>1. Download the Go installation file:</h3>
+    <p>Access the official website or use the command below:</p>
+    <pre><code>wget https://golang.org/dl/go1.23.3.linux-amd64.tar.gz</pre></code>
+    <h3>2. Remove previous Go installations:</h3>
+    <pre><code>sudo rm -rf /usr/local/go</pre></code>
+    <h3>3. Extract the new Go archive to `/usr/local`:</h3>
+    <pre><code>sudo tar -C /usr/local -xzf go1.23.3.linux-amd64.tar.gz</pre></code>
+    <h3>4. Update the PATH by adding `/usr/local/go/bin`:</h3>
+    <p>Edit the `~/.profile` or `/etc/profile` file and add:</p>
+    <pre><code>export PATH=$PATH:/usr/local/go/bin</pre></code>
+    <p>Or run directly in the terminal for the current session:</p>
+    <pre><code>export PATH=$PATH:/usr/local/go/bin</pre></code>
+    <h3>5. Apply PATH changes immediately:</h3>
+    <pre><code>source ~/.profile</pre></code>
+    <h3>6. Check the Go installation and whether it is the supported version:</h3>
+    <pre><code>go version</pre></code>
+    <blockquote><strong>Note:</strong> For Debian/Ubuntu, you can install Go using Snap:
+    <pre><code>sudo snap install --classic go</pre></code></blockquote>
+  </details>
```

Comment:
Honestly, I would just link to the Go install page instead of writing this down here, since that is up-to-date.